src/core/server/saved_objects/migrations/utils.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { mergeWith } from 'lodash';
+import {
+  SavedObjectMigrationContext,
+  SavedObjectMigrationFn,
+  SavedObjectMigrationMap,
+  SavedObjectUnsanitizedDoc,
+} from '../..';
+
+/**
+ * Merges two saved object migration maps.
+ * @public
```

Comment:
Could you add more details on how exactly the maps are merged, i.e when both maps have migrations for the same version

Reply:
Done!